src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -95,99 +80,110 @@
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="4"
+                                                Grid.Column="3"
                                                 Content="{x:Bind ColorEntryAt(11), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="5"
+                                                Grid.Column="4"
                                                 Content="{x:Bind ColorEntryAt(12), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="6"
+                                                Grid.Column="5"
                                                 Content="{x:Bind ColorEntryAt(13), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="7"
+                                                Grid.Column="6"
                                                 Content="{x:Bind ColorEntryAt(14), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
                                 <ContentControl Grid.Row="1"
-                                                Grid.Column="8"
+                                                Grid.Column="7"
                                                 Content="{x:Bind ColorEntryAt(15), Mode=OneWay}"
                                                 ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
                                                 IsTabStop="False" />
+                                <TextBlock Grid.RowSpan="2"
+                                           Grid.Column="8"
+                                           Padding="10,0,10,0"
+                                           HorizontalAlignment="Center"
+                                           VerticalAlignment="Center"
+                                           AutomationProperties.AccessibilityView="Raw"

```

Comment:
I am still pretty weirded out about this. _Why_ does this work for the accessibility tools? What are they seeing that offers them the actual name?

---

it looks like it's the ToString for the item itself... hmm

<img width="291" alt="image" src="https://user-images.githubusercontent.com/189190/198738508-702748ab-0277-4605-859d-81bb3e5018ea.png">

good for now but that probably explains why the `default` tag wasn't working